Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Listview/index.md

```diff
@@ -0,0 +1,115 @@
+---
+versionFrom: 8.4.0
+---
+
+# List View
+
+`Alias: Umbraco.Listview`
+
+`Returns: IEnumerable<IPublishedContent>`
+
+**List View** display a list of category when it is enabled on a documentype that has children.
+
+![List view example](images/listview.png)
+
+### Enable list view
+
+If enabled, editors will be able to see multiple children from a list on a content node that has children otherwise no list will be shown and all children will be shown in content tree.
+
+![Enable List view example](images/enable-listview.png)
+
+## Settings
+
+![List view settings example](images/listview-settings.png)
+![List view settings example 2](images/listview-settings-2.png)
+
+### Page Size
+
+Defines how many child content nodes you want to see per page. This will limit how many content items you will see in your list. If you set it to 5, then only 5 content items will be shown in the list.
+
+### Order By
+
+Will sort your list by the selection you choose in the dropdown. By default it selects "Last edited". By default you get the following three:
+
+- **Last edited** - When the content node was last edited and saved.
+- **Name** - Name of the content node(s).
+- **Created by** - This is the user who the content node was created by.
+
+You can add more sorting to this list by adding more datatypes to the columns in the "Columns Displayed" section.
+
+### Order Direction
+
+You can select order of the content nodes displayed, "Acsending" or "Descending". The order is affected by the "Order By" selection.
+
+### Columns Displayed
+
+It is possible to add more columns to the list, via adding the properties through the dropdown. These properties are based on the datatypes which are used by the document-type. It will show up in the dropdown by its alias and not the name on the property.
```

Comment:
```suggestion
It is possible to add more columns to the list, via adding the properties through the dropdown. These properties are based on the Data Types which are used by the Document Type. It will show up in the dropdown by its alias and not the name on the property.
```